### PR TITLE
ADD pdf reading as an input option

### PR DESCRIPTION
**Pull request by bkolosk1**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -14,3 +14,4 @@ six>=1.16.0
 tqdm>=4.60
 pytest>=7.1.2
 pytest-sugar>=0.9.5
+pymupdf
```
Comment (SkBlaz) [inline requirements.txt:17]: Add exact version please

File: rakun2/class_rakun.py [hunk 2]
```diff
@@ -231,13 +232,21 @@ def parse_input(self, document: str, input_type: str) -> None:
             with open(document, "r", encoding="utf-8") as doc:
                 full_document = doc.read().split("\n")
 
+        if input_type == "pdf":
+            with fitz.open(document) as doc:
```
Comment (SkBlaz) [inline rakun2/class_rakun.py:236]: add encoding

Comment (SkBlaz) [reply rakun2/class_rakun.py:236]: + maybe an example in `examples`?

**Review by SkBlaz — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 1 above)
Comment (bkolosk1) [reply requirements.txt]: Done

File: rakun2/class_rakun.py (hunk 2 above)
Comment (bkolosk1) [reply rakun2/class_rakun.py:235]: Done

**Review by SkBlaz — APPROVED** [review]

**Pull request merged**